linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_encrypt_sent_records/rule.yml

```diff
@@ -31,8 +37,14 @@ ocil_clause: 'audispd is not encrypting audit records when sent over the network
 ocil: |-
     To verify the audispd plugin encrypts audit records off-loaded onto a different
```

Comment:
In latest Fedora and RHEL8 audispd plugin is not used to off-load the records. This paragraph should be within the `else` case, and a paragraph for Fedora and RHEL8 added.

Reply:
This is ok, actually, audispd plugins are used.